validator/tests/test_client_request_handlers/test_submit_handlers.py

```diff
@@ -52,6 +54,7 @@ def test_batch_submit_without_wait(self):
 
         self.assertEqual(self.status.OK, response.status)
 
+    @unittest.expectedFailure
```

Comment:
This should be replaced by a unit test for the preprocessor to ensure deserialization failures are handled correctly.